src/platform/forms-system/test/js/review/ObjectField.unit.spec.jsx

```diff
@@ -904,4 +904,68 @@ describe('Schemaform review: ObjectField', () => {
     fireEvent.click(editButton);
     expect(onClick.called).to.be.true;
   });
+
+  it('should render custom ObjectViewField even when not attached to the root', () => {
+    const onChange = sinon.spy();
+    const onBlur = sinon.spy();
+
+    const schema = {
```

Comment:
To help emphasize that this is the schema for the _entire_ page, but that we're using only part of it to simulate rendering the nested object, you can prefix these variable names with `page`.